manual/manual.html

```diff
@@ -7880,11 +7889,33 @@ <h3 id="_words_identifiers_and_names"><a class="anchor" href="#_words_identifier
 </div>
 </div>
 <div class="paragraph">
-<p>A quoted identifier starts and ends with single quotes and may contain any character except quotes (including spaces).
-By quoting any sequence of characters can become an identifier.
-A quoted identifier may also be used to make an identifier out of a reserved word such as <code>Look</code>.
-This is useful in the definition of the verb <code>look</code>.
-It would look like:</p>
+<p>A quoted identifier starts and ends with single quotes and may contain any character (including spaces) except quotes.
```

Comment:
It should be explicit what kind of quotes the last "quotes" in this sentence refers to. (single).

Reply:
I have kept it vague because it should mean any type of quotes, single OR double — after all, double quotes are not allowed in identifiers.

Do you think this is confusing?

Reply:
I think that to dispel confusion, we should move the tip "Escaping Single Quotes Inside Quoted Identifiers" from the _Names Containing Multiple Words_ subsection (which comes shortly after) to the _Quoted Identifiers_ subsection, so we fully cover the syntax of quoted identifiers in one place.

Then in the _Names Containing Multiple Words_ subsection we could just add a quick reminder or a "(see §xxx)" Xref pointing back to that rule.

It would make more sense to first exhaust all rules of quoted identifiers, and then the following subsections _Stropping_ and  _Names Containing Multiple Words_ are just sections dealing with the different uses of quoted IDs.

Reply:
I was wrong, it's legal to use double quotes inside a quoted ID:

    THE 'xx"xx' IsA object.

the above works fine. I'll amend the text as suggested, specifying the single quote exception, and also mention that single quotes need to be escaped by doubling if required as part of the ID.

Reply:
Ok, commit 3cc0d88 should improve the text on Quoted IDs, and it now includes an explicit mention on how to escape single quotes inside a quoted ID, with a practical example. It also explains how the escaped quote is handled as one single-quote in the printed text inside the game, and that delimiting quotes are ignored in print.

(it's not easy to work on this text because of sentences like "double single quote" and "single single quote" which end up being confusing to read)

Reply:
Good. I agree with the move that you suggested [above](https://github.com/alan-if/alan-docs/pull/55#discussion_r319760163).